www/docs/docusaurus.config.js

```diff
@@ -54,6 +55,19 @@ const config = {
       theme: require("prism-react-renderer/themes/vsDark"),
       darkTheme: require("prism-react-renderer/themes/vsDark"),
     },
+    zoom: {
+      selector: '.markdown img',
+      background: {
+        light: 'rgb(255, 255, 255)',
+        dark: 'rgb(50, 50, 50)'
+      },
+      config: {
+        container: {
+          left: 250,
+          right: 40
+        }
```

Comment:
Why is this necessary? I removed it and nothing changed

Reply:
I had done this to zoom in the image inside the text container and not overflow over the sidebar, but I will make the appropriate changes in the z-index